stl/src/filesys.cpp

```diff
@@ -379,10 +375,10 @@ _FS_DLL int __CLRCALL_PURE_OR_CDECL _Equivalent(
     } else if (!_Ok1 || !_Ok2) {
         return 0;
     } else { // test existing files for equivalence
-        return _Info1.dwVolumeSerialNumber != _Info2.dwVolumeSerialNumber
-                    || _Info1.nFileIndexHigh != _Info2.nFileIndexHigh || _Info1.nFileIndexLow != _Info2.nFileIndexLow
-                 ? 0
-                 : 1;
+        return _Info1.dwVolumeSerialNumber == _Info2.dwVolumeSerialNumber

```

Comment:
here as well only `memcmp` can be used

Reply:
Not really, since we are only comparing three fields, and these are not all of them